definitions/mock/ibc.mock.proto

```diff
@@ -1,19 +1,25 @@
 syntax = "proto3";
 package ibc.mock;
 
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
 import "ibc/core/client/v1/client.proto";
 
 message Header {
   ibc.core.client.v1.Height height = 1;
-  uint64 timestamp = 2; 
+  google.protobuf.Timestamp timestamp = 2;
 }
 
 message ClientState {
   Header header = 1;
+  google.protobuf.Duration trusing_period = 2;
+  google.protobuf.Duration max_clock_drift = 3;
+  bool frozen = 4;
 }
 
 message ConsensusState {
   Header header = 1;
+  bytes bytes = 2;
```

Comment:
Renaming the `bytes` field to `root`? Given that consensus states usually track the state machine by this field. (Although I acknowledge that solomachine, for example, does not have this)

Reply:
hmm. ok. but I am realizing we should specify the behavior of the Mock host.

Reply:
and leads to another question - should we even bother with a Mock host - when we can just have tm host ?